pkg/rkt/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.12.4-alpine
+
+RUN apk add ca-certificates gcc libc6-compat gnupg make automake wget git coreutils cpio squashfs-tools autoconf file xz patch bc acl-dev openssl-dev at-spi2-core python pkgconfig zlib-dev musl-dev pixman libcap libfdt bash
+
+RUN git clone https://github.com/rkt/rkt.git
+RUN cd rkt && git reset --hard tags/v1.26.0
```

Comment:
Instead of cd please use WORKDIR /rkt Dockerfile directive

Reply:
Fixed.